package.json

```diff
@@ -103,7 +103,7 @@
     "mock-express-response": "^0.1.2",
     "nock": "^9.0.0",
     "node-mocks-http": "^1.5.3",
-    "nyc": "^9.0.1",
+    "nyc": "^6.6.0",
```

Comment:
Tou need to update the `yarn.lock` file as well or this won't actually downgrade Yarn